use super::GlobalError;

pub fn run() -> Result<(), GlobalError> {
    Ok(())
}

#[test]
fn challenge_one_day_13() -> Result<(), GlobalError> {
    assert_eq!(2, 2);
    Ok(())
}
#[test]
fn challenge_two_day_13() -> Result<(), GlobalError> {
    assert_eq!(2, 2);
    Ok(())
}
